Formula/gst-python.rb

```diff
@@ -10,21 +10,51 @@ class GstPython < Formula
     sha256 "e708aa675d5744fc36608249728290d6d97803cd071bf2ac91d245a9d8e8c996" => :mavericks
   end
 
+  option "without-python", "Build without python 2 support"
```

Comment:
If I understood @tdsmith correctly (I'm not really familiar with Python), `depends_on :python` will create a hard dependency on Homebrew-provided Python (for bottle users). Adding just the `option` will allow the formula to build against system Python.